src/Command/MarkMigrated.php

```diff
@@ -60,14 +61,33 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption(
+                'target',
+                'u',
+                InputArgument::OPTIONAL,
+                'If present it will mark migrations from beginning to the given one'
+            )
+            ->addOption(
+                'exclude',
+                't',
```

Comment:
Well, as @dakota pointed out, it would make sense to have the same behaviour for all commands to have the consistency. It's less surprising for the end user and someone who is new to the tool. 

So @dakota's comment is valid (https://github.com/cakephp/migrations/pull/137#discussion_r41730174).

To sum up :

`bin/cake migrations mark_migrated` should migrate every migrations that has not been migrated
`bin/cake migrations mark_migrated all` should have the same effect (and show a deprecation warning)

`bin/cake migrations mark_migrated --target=20150417223600 ` should migrate all migrations up to the target param.
`bin/cake migrations mark_migrated 20150417223600` should have the same effect (and show a deprecation warning)

`bin/cake migrations mark_migrated --target=20150417223600 --exclude ` should migrate all migrations up to the target param, excluding it.

Is that ok with you @dakota ?

Sorry for the back and forth @pedrofs